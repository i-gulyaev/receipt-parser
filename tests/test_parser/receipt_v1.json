{"user":"ОБЩЕСТВО С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ \"ВКУСВИЛЛ\"","userInn":"7734675810  ","requestNumber":14,"shiftNumber":199,"operator":"АУР","operationType":1,"totalSum":33900,"cashTotalSum":0,"ecashTotalSum":33900,"kktRegId":"0004069170019909    ","fiscalDriveNumber":"9280440301335384","fiscalDocumentNumber":15504,"fiscalSign":3355941482,"items":[{"name":"Сыр козий \"Валансе\" с белой плесенью в золе,шт","price":33900,"sum":33900,"quantity":1.0,"paymentType":4,"productType":1,"nds":2}],"nds10":3082,"code":3,"fiscalDocumentFormatVer":2,"machineNumber":"00106309178411","retailPlace":"Магазин продуктов для здорового питания","prepaidSum":0,"creditSum":0,"provisionSum":0,"dateTime":1620799500,"taxationType":1,"localDateTime":"2021-05-12T09:05"}