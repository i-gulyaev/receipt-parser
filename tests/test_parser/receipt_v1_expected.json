{
  "seller": "ОБЩЕСТВО С ОГРАНИЧЕННОЙ ОТВЕТСТВЕННОСТЬЮ \"ВКУСВИЛЛ\"",
  "sum": 339,
  "items": [
    {
      "name": "Сыр козий \"Валансе\" с белой плесенью в золе,шт",
      "price": 339,
      "sum": 339,
      "quantity": 1
    }
  ],
    "date": "2021-05-12T09:05:00"
}
